### ISSUE TITLE
JavaScript heap out of memory

### ISSUE THREAD

**Issue by abepuentes** [dabit3/next.js-amplify-workshop#5]
Greetings,
Great tutorial as usual by Naber, so thankfull.

I wonder if any one else has been having this issue though. Error happend when I was half throw the tutorial, just in dev mode. 
I know this has nothing to do with this tutorial I'm just posting to know if it did happend to someone else, seems like a amplify console related issue.

OS: Windows 10.
Editor: VSCode
App: NextJS, TailwindCSS
Node 14LTS

<--- Last few GCs --->

[46644:000001875BA2FE60]  7999109 ms: Mark-sweep (reduce) 4078.1 (4104.2) -> 4077.5 (4103.7) MB, 3561.1 / 0.0 ms  (average mu 
= 0.096, current mu = 0.000) allocation failure scavenge might not succeed
[46644:000001875BA2FE60]  8000736 ms: Mark-sweep (reduce) 4078.5 (4100.7) -> 4078.3 (4102.4) MB, 1625.7 / 0.1 ms  (average mu 
= 0.063, current mu = 0.001) allocation failure scavenge might not succeed

<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF64F09021F napi_wrap+109311
 2: 00007FF64F035286 v8::internal::OrderedHashTable<v8::internal::OrderedHashSet,1>::NumberOfElementsOffset+33302
 3: 00007FF64F036056 node::OnFatalError+294
 4: 00007FF64F90054E v8::Isolate::ReportExternalAllocationLimitReached+94
 5: 00007FF64F8E53CD v8::SharedArrayBuffer::Externalize+781
 6: 00007FF64F78F85C v8::internal::Heap::EphemeronKeyWriteBarrierFromCode+1516
 7: 00007FF64F79AB9A v8::internal::Heap::ProtectUnprotectedMemoryChunks+1258
 8: 00007FF64F797D49 v8::internal::Heap::PageFlagsAreConsistent+2457
 9: 00007FF64F78C971 v8::internal::Heap::CollectGarbage+2033
10: 00007FF64F78AB75 v8::internal::Heap::AllocateExternalBackingStore+1317
11: 00007FF64F7AAF67 v8::internal::Factory::NewFillerObject+183
12: 00007FF64F4DAFF1 v8::internal::interpreter::JumpTableTargetOffsets::iterator::operator=+1409
13: 00007FF64F988EFD v8::internal::SetupIsolateDelegate::SetupHeap+463949
14: 000000EF75E8499F 
error Command failed with exit code 134.

**Comment by dabit3** [comment]
Hey @abepuentes , thanks for the issue. I tried to reproduce but did not have any luck. I am on a mac machine though so I wonder if this issue is OS dependent?

**Comment by abepuentes** [comment]
Yes @dabit3, it's seems to be related to windows. Actually, I've found the same error report also on others windows machines using amplify console [https://github.com/aws-amplify/amplify-console/issues/1180](here). That been said, I was able to continue the workshop after restarting vscode, and I want to thanks again for your work becuase has been extremely helpfull and usefull.
